front/app/modules/commercial/admin_project_templates/index.tsx

```diff
@@ -1,10 +1,14 @@
 import React, { ReactNode } from 'react';
 import { ModuleConfiguration } from 'utils/moduleUtils';
-import CreateProjectFromTemplate from './admin/containers/CreateProjectFromTemplate';
-import Tab from './admin/components/Tab';
+const CreateProjectFromTemplate = React.lazy(
```

Comment:
All the import changes were done automatically via a script so good to have a sanity check on them.